db/schema.rb

```diff
@@ -124,6 +124,7 @@
     t.integer "previous_id"
     t.boolean "is_published", default: false
     t.string "author"
+    t.boolean "navigation", default: true
```

Comment:
I don't think this should be here - is this not from a previous attempt at doing article pages?